composer.json

```diff
@@ -17,16 +18,51 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "zendframework/zendframework": "2.*",
-        "phly/phly-common": "dev-master"
+        "php": "^7.3",
+        "laminas/laminas-console": "^2.5",
+        "laminas/laminas-dependency-plugin": "^1.0",
+        "laminas/laminas-eventmanager": "^3.0",
+        "laminas/laminas-feed": "^2.5",
+        "laminas/laminas-filter": "^2.5",
+        "laminas/laminas-http": "^2.5",
+        "laminas/laminas-i18n": "^2.5",
+        "laminas/laminas-inputfilter": "^2.5",
+        "laminas/laminas-modulemanager": "^2.5",
+        "laminas/laminas-mvc": "^3.0",
+        "laminas/laminas-paginator": "^2.5",
+        "laminas/laminas-stdlib": "^3.0",
+        "laminas/laminas-tag": "^2.5",
+        "laminas/laminas-uri": "^2.5",
+        "laminas/laminas-validator": "^2.5",
+        "laminas/laminas-view": "^2.5",
+        "phly/phly-common": "dev-refresh"
```

Comment:
This line and the `repositories` block will be removed after `PhlyCommon` was merged and tagged.